script/10_deployMultiFeedAdaptersWithoutRounds.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+echo "Instructing the shell to exit immediately if any command returns a non-zero exit status..."
+set -e
+echo "Done."
+
+echo "Navigating to the root directory of the project..."
+cd ../
+echo "Done."
+
+echo "Setting environment variables..."
+source .env
+echo "Done."
+
+echo "Creating $NETWORK directory inside deployment/artifacts/contracts directory..."
+if [ -z "$NETWORK" ]
+then
+      echo "NETWORK variable inside .env file is not set. Please set NETWORK environment variable."
+      exit 1
+else
+      if [ -d "deployment/artifacts/contracts/$NETWORK" ]
+      then
+            echo "Directory deployment/artifacts/contracts/$NETWORK already exists."
+      else
+            mkdir deployment/artifacts/contracts/$NETWORK
+      fi
+fi
+echo "Done."
```

Comment:
Yes, that's correct and it presents an opportunity for code simplification and reduction.

But given that it would impact a large number of existing files, I don’t see enough benefit to make the changes at this time.